test/test_suite.py

```diff
@@ -1008,6 +1009,10 @@ def test_text_roundtrip(self):
         eq_(row, ("some text",))
 
 
+@pytest.mark.skipif(
+    os.getenv("SPANNER_EMULATOR_HOST"),
+    reason="Numeric type isn't supported by emulator yet",
+)
```

Comment:
I don't think these should fail. The latest release of the emulator does support NUMERIC. See https://github.com/googleapis/nodejs-spanner/pull/1358 where we enabled the tests on Node.js.